test/python/circuit/test_parameters.py

```diff
@@ -1106,6 +1106,7 @@ def test_circuit_with_ufunc(self):
         theta = Parameter(name="theta")
 
         qc = QuantumCircuit(2)
+        qc.p(numpy.abs(phi), 0)
```

Comment:
These changes seem unrelated to the `abs` implementation 🤔

Reply:
I thought this will test whether `abs` can be used when constructing the circuit, but this may not be necessary...

Reply:
Oh no my bad, with all the trigonometric functions I misread `abs` here 😄